### PR TITLE
refactor: replaced wait instruction with serviceConfig.readyConditions config

### PR DESCRIPTION
**Pull request by leoporoli**

### CODE REVIEW

File: src/cl/cl_node_ready_conditions.star [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    recipe = GetHttpRequestRecipe(
+        endpoint = "/eth/v1/node/health",
+        port_id = port_id
+    )
+
+    ready_conditions = ReadyCondition(
+        recipe = recipe,
+        field = "code",
+        assertion = "IN",
+        target_value = [200, 206, 503],
```
Comment (adschwartz) [inline src/cl/cl_node_ready_conditions.star:11]: Did you mean to leave `503` in here? If so why would that be considered a valid "ready code"?

**Review by adschwartz — CHANGES_REQUESTED** [review]
@leoporoli looks good, but not sure about that `503` in the ready condition

**Comment by leoporoli** [comment]
> @leoporoli looks good, but not sure about that `503` in the ready condition
now it's fixed, more context in this comment: https://github.com/kurtosis-tech/eth-network-package/pull/19#discussion_r1194366600

**Comment by leoporoli** [comment]
@adschwartz don't forget this guy

**Review by adschwartz — APPROVED** [review]

**Pull request merged**